crates/core/transaction/src/gas.rs

```diff
@@ -47,9 +48,7 @@ pub fn spend_gas_cost() -> Gas {
 
 pub fn output_gas_cost() -> Gas {
     Gas {
-        // Each [`Action`] has a `0` `block_space` cost, since the [`Transaction`] itself
-        // will use the encoded size of the complete transaction to calculate the block space.
-        block_space: 0,
+        block_space: NOTEPAYLOAD_SIZE + 352,
```

Comment:
Not sure where this came from since there's no comment, but I count (ignoring proto framing)

NOTEPAYLOAD_SIZE + balance_commitment (32 bytes) + wrapped_memo_key (48 bytes) + ovk_wrapped_key (48 bytes) + zk proof (192) 

which would be + 320 rather than + 352.

The proto message for OutputBody has a comment saying the ovk_wrapped_key is 80 bytes but this seems to be incorrect from looking at the source code and should probably be corrected: https://buf.build/penumbra-zone/penumbra/docs/main:penumbra.core.component.shielded_pool.v1#penumbra.core.component.shielded_pool.v1.OutputBody